docs/fundamentals/actions.md

```diff
@@ -56,49 +56,27 @@ showModule(Module.DASHBOARD);
 
 ## Payloads
 
-Care should be taken when designing the shape of an action payload. In the above
-example the payload is a simple enum, but there will be cases where more 
-complex structures are needed.
+Care should be taken when designing the shape of an action payload. In the above example, the payload is a simple enum, but there will be cases where more complex structures are needed.
 
 ### Actions should describe *what* is happening
 
-We recommend trying to treat actions more as "describing events that occurred",
-rather than "setters". Treating actions as "events" generally leads to more
-meaningful action names, fewer total actions being dispatched, and a more
-meaningful action log history. Writing "setters" often results in too many
-individual action types, too many dispatches, and an action log that is less
-meaningful.
+We recommend trying to treat actions more as "describing events that occurred", rather than "setters". Treating actions as "events" generally leads to more meaningful action names, fewer total actions being dispatched, and a more meaningful action log history. Writing "setters" often results in too many individual action types, too many dispatches, and an action log that is less meaningful.
 
-Furthermore, "describing events that occurred" can prevent coupling between
-modules. A module should know how to describe an event that happens through an
-action, but it shouldn't care about how that action is executed (i.e. a certain
-order of action setters).
+Furthermore, "describing events that occurred" can prevent coupling between modules. A module should know how to describe an event that happens through an action, but it shouldn't care about how that action is executed (i.e. a certain order of action setters).
 
 #### Batch actions
 
-If you find code that calls the same action multiple times right after each
-other, the payload is badly designed. This is a symptom that the callsite tries
-to use the actions as instructions for how the app should transition its state,
-rather than telling the app what the change is. In these cases, the payload 
-should be designed so it can contain, for example, changes to multiple entities
-at once.
+If you find a code that calls the same action multiple times right after each other, the payload is badly designed. This is a symptom that the call site tries to use the actions as instructions for how the app should transition its state, rather than telling the app what the change is. In these cases, the payload should be designed so it can contain, for example, changes to multiple entities at once.
```

Comment:
"a code" sounds weird to me. Here I would keep code is an uncountable noun.

Reply:
change to `a piece of code` | `a part of code`?

Reply:
updated to the former

Reply:
I'd delete `a` as Øyvind suggested

Reply:
Reverted to just `code`.